src-tauri/src/farmer.rs

```diff
@@ -203,12 +234,49 @@ async fn farm(
     )
     .await?;
 
-    Ok(multi_plots_farm)
-}
+    // Start RPC server
+    let ws_server = match WsServerBuilder::default()
+        .build(ws_server_listen_addr)
+        .await
+    {
```

Comment:
Why do you need farmer's RPC server?

Reply:
we don't. I was not putting it in here since farmer DSN is not ready till now. This time I put it in because every update I have to delete all the stuff related to farmer dsn in here, and thought if it won't hurt, let's include it, it will make the updates easier.

On the other hand, if you suggest to remove it, im fine with it, I can remove them